bindings/jupyter/mlvis/widget_ext/Manifold.py

```diff
@@ -23,10 +21,11 @@ def __init__(self, props={}):
             else:
                 raise exception
 
-        props['data']['x'] = self.process_x(props['data']['x'])
-        props['data']['yPred'] = self.process_y_pred(props['data']['yPred'])
-        props['data']['yTrue'] = self.process_y_true(props['data']['yTrue'])
-        super(Manifold, self).__init__(props)
+        processed_props = props.copy()
```

Comment:
mind adding an inline comment on what the .copy() operation? for future reference when editing this part of the code.